### PR TITLE
Development

### PR DESCRIPTION
**Pull request by abinthomas258**

### CODE REVIEW

File: src/pages/dashboard/components/header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            onKeyDown={showAppBar}
+          >
+            <div className="app-bar"></div>
+            <div className="app-bar"></div>
+            <div className="app-bar"></div>
+          </div>
+          <div className="box-container flex justify-between items-center">
+            <div className="brand-banner">myStays.com</div>
+            <div className="flex justify-between items-center mr-7 ">
+              <svg
```
Comment (abhijithramankutty) [inline src/pages/dashboard/components/header/Header.tsx:29]: svg image can be moved to a separate .svg file and import as an image here

File: src/pages/dashboard/components/navbar/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+export const NavBar = ({ display }: NavBarProps) => {
+  return (
+    <>
+      <nav className="navbar" style={{ display: display }}>
+        <ul className="navbar-nav">
+          <li className="nav-item">
+            <div className="nav-link">
+              <NavLink to="/" className="link-text flex items-center">
+                <div>
+                  <svg
```
Comment (abhijithramankutty) [inline src/pages/dashboard/components/navbar/NavBar.tsx:16]: svg image can be moved to a separate .svg file and import as an image here

File: src/pages/sign-up/SignIn.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+function SignUp() {
+  return <h1>SignUp</h1>
+}
+
+export default SignUp
```
Comment (abhijithramankutty) [inline src/pages/sign-up/SignIn.tsx:7]: change the component file name also to SignUp

File: src/pages/dashboard/components/navbar/NavBar.tsx (hunk 2 above)
Comment (abinthomas258) [reply src/pages/dashboard/components/navbar/NavBar.tsx]: done

File: src/pages/sign-up/SignIn.tsx (hunk 3 above)
Comment (abinthomas258) [reply src/pages/sign-up/SignIn.tsx]: done

File: src/pages/dashboard/components/header/Header.tsx (hunk 1 above)
Comment (abinthomas258) [reply src/pages/dashboard/components/header/Header.tsx]: done

**Review by abhijithramankutty — APPROVED** [review]

**Comment by kalarikkalbabu** [comment]
The tests have failed. Please check the logs and fix the issues.

**Pull request merged**